test/__snapshots__/build-css.jest.js.snap

```diff
@@ -639,13 +645,34 @@ legend{
   height:20px;
   margin-top:0;
   border-radius:50%;
-  border:2px solid currentColor;
+  border:1px solid currentColor;
   background:#fff;
-  cursor:pointer;
+  cursor:grab;
+}
+
+.range > input::-webkit-slider-thumb:active{
+  cursor:grabbing;
+}
+
+.range > input::-moz-range-thumb:active{
+  cursor:grabbing;
+}
+
+.range > input::-ms-thumb:active{
+  cursor:grabbing;
```

Comment:
For some reason, the cursor was not being updated when I used a comma-separated list of selectors -- I'm not sure why this is the case.